applications/GeoMechanicsApplication/custom_conditions/axisymmetric_U_Pw_normal_face_load_condition.cpp

```diff
@@ -46,11 +46,31 @@ template< unsigned int TDim, unsigned int TNumNodes >
 GeometryData::IntegrationMethod 
     AxisymmetricUPwNormalFaceLoadCondition<TDim,TNumNodes>::GetIntegrationMethod() const
 {
-    return GeometryData::IntegrationMethod::GI_GAUSS_2;
+    GeometryData::IntegrationMethod GI_GAUSS;
+    //
+    switch (TNumNodes) {
+    case 2:
+        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_2;
+        break;
+    case 3:
+        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_2;
+        break;
+    case 4:
+        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_3;
+        break;
+    default:
+        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_2;
+        break;
+    }
```

Comment:
This is probably my lack of knowledge, but this switch is not completely clear to me. We added <2,3>, <2,4> and <2,5> elements, which now result in `GI_GAUSS_2`, `GI_GAUSS_3` and `GI_GAUSS_2` respectively. Why is that?

Reply:
Gauss points. 2 noded element, needs 1 point. However, we use usually 2 points practically we do not get accurate solution wih 1 point.
For 3 noded => 2 points
4 noded => 3 points
5 noded => proposed to give by 4 or 5 points. However, 2 points gave accurate solution, why 4 or 5 points gave a difference of 10^-8. It needs to be investigated, but I think it has no priority at moment.

Reply:
I do not have enough knowledge about this topic to know whether 5 noded elements with a 2 point gauss integration method are common/feasible for now. Do you know @WPK4FEM?

Reply:
It is fixed, I found a node ordering mistake in the input file. The code is fixed:

    switch (TNumNodes) {
    case 2:
    case 3:
        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_2;
        break;
    case 4:
        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_3;
        break;
    case 5:
        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_5;
        break;
    default:
        GI_GAUSS = GeometryData::IntegrationMethod::GI_GAUSS_2;
        break;
    }